pyspecProcScripts/phasing.py

```diff
@@ -320,6 +321,8 @@ def filter_ranges(B, A):
     peakrange = peakrange[0]
     frq_center = np.mean(peakrange).item()
     frq_half = np.diff(peakrange).item() / 2
+    d.set_prop(peakrange, frq_half)
```

Comment:
The first argument should be a string. This basically gets loaded into a dictionary.

Reply:
In other words, the first argument is like a key